### ISSUE TITLE
[BUG] Serverside error index local 'Owner' (a nil value)

### ISSUE THREAD

**Issue by wrefgtzweve** [Stooberton/ACF-3#329]
I'm not sure how to replicate this as i saw it in console, it happened quite a few times. I'd assume this happened when someone used reload with the armor tool on a world door, I've tried to replicate it but i couldn't.

Stacktrace:
```lua
addons/acf-3/lua/weapons/gmod_tool/stools/acfarmorprop.lua:475: attempt to index local 'Owner' (a nil value)
   1.  __index - [C]:-1
    2.  GetResult - addons/acf-3/lua/weapons/gmod_tool/stools/acfarmorprop.lua:475
     3.  Reload - addons/acf-3/lua/weapons/gmod_tool/stools/acfarmorprop.lua:567
      4.  unknown - gamemodes/sandbox/entities/weapons/gmod_tool/shared.lua:273
```
Locals:
```
ConNum      = Number [0]
Count       = Number [0]
PhysTotal   = Number [0]
Class       = "prop_door_rotating"
Fuel        = Number [0]
OtherNum    = Number [1]
Constraints = {
}
ParNum      = Number [0]
```

**Comment by TwistedTail** [comment]
Probably just a prop protection addon making CPPI return nil when the owner is the world.

**Comment by TwistedTail** [comment]
Apparently I never closed this one, should be fixed on 5e41f0a.
